localstack-core/localstack/services/ec2/provider.py

```diff
@@ -539,6 +542,33 @@ def create_flow_logs(
 
         return response
 
+    @handler("GetSecurityGroupsForVpc", expand=False)
+    def get_security_groups_for_vpc(
+        self,
+        context: RequestContext,
+        get_security_groups_for_vpc_request: GetSecurityGroupsForVpcRequest,
+    ) -> GetSecurityGroupsForVpcResult:
+        vpc_id = get_security_groups_for_vpc_request.get("VpcId")
+        if vpc_id:
+            next_token = get_security_groups_for_vpc_request.get("NextToken", None)
```

Comment:
`NextToken` is used for pagination, which is not implemented for this operation. A null value must be returned.